README.md

```diff
@@ -13,6 +13,7 @@
 ![StandardRB](https://github.com/hopsoft/stimulus_reflex/workflows/StandardRB/badge.svg)
 ![Tests](https://github.com/hopsoft/stimulus_reflex/workflows/Tests/badge.svg)
 [![Discourse](https://img.shields.io/badge/Forum-Discourse-blue.svg?style=flat)](https://stimulus-reflex.discourse.group)
+[![Discord](https://img.shields.io/badge/discord-join-7289DA.svg?logo=discord&longCache=true&style=flat)](https://discord.gg/XveN625)
```

Comment:
Will you update the badge to https://img.shields.io/badge/Chat-Discord-blue.svg?style=flat